integration_test/third_party_apps_test.go

```diff
@@ -848,9 +909,16 @@ func TestThirdPartyApps(t *testing.T) {
 	tests := []test{}
 	allApps := fetchAppsAndMetadata(t)
 	platforms := strings.Split(os.Getenv("PLATFORMS"), ",")
+
 	for _, platform := range platforms {
 		for app, metadata := range allApps {
-			tests = append(tests, test{platform: platform, app: app, metadata: metadata, skipReason: ""})
+			if gpus, ok := gpuApps[app]; ok {
```

Comment:
It might be too much trouble, but I'd rather see this be driven by the apps' metadata.yaml files instead of hardcoded here. If that's not easy right now, this file's already pretty complicated so maybe just leave a TODO to push this into metadata.yaml in the future...

Reply:
Done! Added the `gpu_models` option to the metadata.yaml.